.gitignore

```diff
@@ -89,7 +89,4 @@ public
 .fusebox/
 
 # DynamoDB Local files
-.dynamodb/
-
-# Built library shouldn't be committed
-lib/
+.dynamodb/
```

Comment:
what is .dynamodb ?  

this looks like you're installing this package, but you should in addition be installing quagga2 ... 

this .d.js and such are part of the quagga package. .. so you should get those *with* this...